src/app/user/login/login.component.ts

```diff
@@ -7,10 +7,22 @@ import { Component, OnInit } from '@angular/core';
 })
 
 export class LoginComponent implements OnInit {
-  hide = true
+  hide = true;
+  loginError = false;
+
+  testEmail = ["rtoche@uark.edu"]
+  testPassword = ["legomyego"]
+
   constructor() { }
 
   ngOnInit(): void {
   }
 
+  login(userEmail: string, userPassword: string):void {
+    console.log(userEmail);
+    console.log(userPassword);
+    this.loginError = true;
```

Comment:
is this just being set to true to test the error message? if so, just resolve this comment, it makes sense to me

Reply:
Yes for testing purposes